src/style.css

```diff
@@ -0,0 +1,87 @@
+html {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Aventir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+body {
+  margin-top: 20px;
+  margin-left: 48px;
+
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.big {
+  box-sizing: border-box;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  width: 100%;
+  height: 70px;
+  margin-bottom: 20px;
+}
+
+.small {
+  box-sizing: border-box;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  width: 100%;
+  height: 42px;
+}
+
+.textBig {
+  padding-left: 62px;
+}
+
+.textSmall {
+  padding-left: 33px;
+}
+
+.img:hover {
+  color: red;
+}
+
+.img:focus {
+  color: aqua;
+}
```

Comment:
```suggestion
```

there are no such colors in the design + the image doesn't have text content to apply `color` on 😉